version bump to 19.02.28, Bug 683684, fix Bug 652260, update manifest

donahue95 opened the pull request.

a17r commented:
There is no manifest for ebuilds in a long time. How did you create that bump, don't you work on a git clone?

donahue95 commented:
I have a local overlay in which I copied, edited, and renamed the shotcut-
19.01.24.ebuild to shotcut-19.02.28.ebuild. I then updated the manifest and ran
emerge shotcut and then tested shotcut-19.02.28 satisfactorily. I then synced my
local machine (clone) gentoo repository and my donahue95/gentoo fork of
gentoo/gentoo from the bash terminal. I then edited the mainifest file in the
local machine repo to the content of the overlay manifest and copied the
shotcut-19.02.28.ebuild file from the overlay to the local machine repo. I then
ran git add . and git commit -m and git push origin master. I then browsed to
github.com/donahue95/gentoo and generated a pull request. The bot response to
the pull request made clear that my commit message was deficient in not having a
signed off by entry and properly formatted Bug links. I used git commit --amend
and git push --force origin master to improve the commit message. Added [request
reassignment] to the pull request title at gentoo/gentoo. 
Never ever submit a manifest with a package change? 
On Wed, 2019-05-08 at 09:20 -0700, Andreas Sturmlechner wrote:
> There is no manifest for ebuilds in a long time. How did you create that bump,
> don't you work on a git clone?
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.copiied
